benchmarks/dynamo/timm_models.py

```diff
@@ -296,7 +296,8 @@ def _gen_target(self, batch_size, device):
     def compute_loss(self, pred):
         # High loss values make gradient checking harder, as small changes in
         # accumulation order upsets accuracy checks.
-        return self.loss(pred, self.target) / 10.0
+        loss = self.loss(pred, self.target) / 10.0
+        return torch.min(loss, torch.tensor(0.5))
```

Comment:
wait, but here you quite possibly sending a zero gradient to backprop? If loss is larger than 0.5 it'll get a zero gradient in backward, and gradient checks would become pointless. 
Also, the loss value itself in many cases should not matter, as the value of the gradient that will be back-propagated won't depend on the loss itself?

Reply:
You are right. Thats just egregious.